spec/requests/posts_controller_spec.rb

```diff
@@ -1,35 +1,35 @@
-# require 'rails_helper'
+require 'rails_helper'
 

```

Comment:
- [ ] Kindly make sure that your API `authenticate` endpoint is able to return an error when the user does not exit in the DB. 
- [ ] Also you are having a syntax error in the  API `authenticate` endpoint. 

![image](https://user-images.githubusercontent.com/65068771/163689564-752706dc-c723-474f-9945-873ec4be2aea.png)

Hint Example :point_down: 

```.rb
module Api
  module V1
    class AuthenticationController < ApplicationController
      rescue_from ActionController::ParameterMissing, with: :parameter_missing

      def create
        params.require(:password)

        user = User.find_by(email: params.require(:email))

        if user 
          token = AuthenticationTokenService.call(user.id)
          render json: { token: token }, status: :created
        else 
           render json: { error: "User does not exist" }
        end

      end

      private

      def parameter_missing(error)
        render json: { error: error.message }, status: :unprocessable_entity
      end
    end
  end
end
```